03_launch_mgmt_cluster.sh

```diff
@@ -378,7 +378,7 @@ function launch_cluster_api_provider_metal3() {
     nohup make run >> capm3.out.log 2>> capm3.err.log &
   fi
 
-  if [ "${BMO_RUN_LOCAL}" == true ] && [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
+  if [ "${BMO_RUN_LOCAL}" == true ] && [ "${CAPM3_VERSION}" == "v1alpha4" ] || [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
```

Comment:
I think we don't even need the CAPM3_VERSION check here , because this was here for v1alpha3 only

Reply:
I agree with you, `if [ "${BMO_RUN_LOCAL}" == true ]` is enough.

Reply:
Done